Servers/UI/OJS.Servers.Ui/ClientApp/.env.production

```diff
@@ -0,0 +1,2 @@
+VITE_UI_SERVER_URL = 'http://217.174.159.212:5010'
+VITE_ADMINISTRATION_URL = 'http://217.174.159.212:5001'
```

Comment:
Shouldn't this be https://alpha.judge.softuni.org and admin.alpha...

Reply:
I think this is the Ip of the https://alpha.judge.softuni.org/

![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/77450034/4d1cde09-ac95-4233-9cc9-f8b383537d00)

Reply:
Done